EventEnabledInsurance/readme.md

```diff
@@ -559,3 +559,64 @@ DB_writer bar file: Responsible for Reading messages from the Queue `Quote` and
 
 # Testing the POST calls via APIC
 Instructions to load test the POST call via APIC can be found [here](post-load-test-readme.md).
+
+# Component Downtime Testing
+
+Prereqs:
+1. Configure kafka connectors
+2. Call REST endpoint post & get
+
+## I. Shutting down the db writer integration server
+
+1. Delete the integration server:
+    ```sh
+    oc -n $NAMESPACE get integrationserver ace-db-writer-int-srv-eei -o yaml > ~/dbwriter.yaml
+    oc -n $NAMESPACE delete integrationserver ace-db-writer-int-srv-eei
+    ```
+    The post call will succeed but the message won't be taken off the queue and won't be processed
+2. Recreate integration server:
+    ```sh
+    bash oc apply -f ~/dbwriter.yaml
```

Comment:
Is bash needed here, maybe just the oc command?

Reply:
it isn't, good catch 👍